src/components/GameCards/GameCard.js

```diff
@@ -39,6 +39,10 @@ export default {
       return this.notes && this.notes[this.gameId] && this.notes[this.gameId].text;
     },
 
+    progress() {
+      return this.progresses && this.progresses[this.gameId] && this.progresses[this.gameId].number;
```

Comment:
`this.progress` is always going to be `true` since it's defined as an empty object. If you want to check if there are any progresses available you could use something like `Object.keys(this.progress).length > 0` or remove it (I suggest removing it).

```suggestion
      return this.progresses[this.gameId] && this.progresses[this.gameId].number;
```
Also, if you want to make the progress platform-specific you'll eventually have to get the platform-specific progress, something like `this.progresses[this.platfor.code][this.gameId].number`